lib/search/stream-filter.js

```diff
@@ -0,0 +1,10 @@
+var ms = require('mississippi')
+
+module.exports = function (filter) {
```

Comment:
This is such a baseline tool that I wanted to hide the implementation but it wasn't worth adding `stream-filter` as a dependency. I guess I can just define it as a utility in `search.js`